keyboards/dztech/dz68rgb/readme.md

```diff
@@ -0,0 +1,16 @@
+# DZ68RGB
+
+A customizable 68% RGB keyboard.
```

Comment:
Hi @moyi4681 I think there are some terminology issues here. 

This is actually what the community calls a 65 "percent" layout. People use 68 to describe 68 "key" layout like the Varmilo VA68m or the Magicforce 68. 

The rest of the boards in this directory seem to be dz(percent)rgb. Perhaps this board should be called a dz65rgb?

Reply:
For what it may be worth, I agree 100% with mechmerlin on this point.